### ISSUE TITLE
[PoS] Add fullNotifications and extendedNotifications in settings

### ISSUE THREAD

**Issue by Janaka-Steph** [btcpayserver/btcpayserver#783]
**Is your feature request related to a problem? Please describe.**
I am using the PoS with callback notification directed to a IOT board that will trigger an action on 1006 invoice_completed IPN. Receiving multiple simultaneous calls is difficult to manage properly and can easily cause the board to be unstable. 

**Describe the solution you'd like**
I would like you to consider adding fullNotifications and extendedNotifications to the PoS settings so I can disable it and receive just one notification after invoice completion.

**Comment by NicolasDorier** [comment]
It seems easy enough.
@Janaka-Steph is it still a problem? Maybe we can add an "advanced" tab in the PoS settings with checkboxes to configure this.

**Comment by Janaka-Steph** [comment]
Hello,
I decided to make a PoS terminal from scratch so I didn't try recently. I may try again to make my vending machine project to work with BTCPay, but it's not a priority for me. 
However, the Lua code I made to power the vending machine that would be triggered by BTCPay is kept on this [repo](https://github.com/bitcoin-studio/LightningNespressoServer), if someone is interested. 
Feel free to close the issue for now.
